hack/test-e2e-provider-local.sh

```diff
@@ -30,7 +30,7 @@ cd $repo_root
 
 version=$(git rev-parse HEAD)
 make docker-images
-docker tag eu.gcr.io/gardener-project/gardener/extensions/networking-cilium:latest networking-cilium-local:$version
+docker tag europe-docker.pkg.dev/gardener-project/releases/extensions/networking-cilium:latest networking-cilium-local:$version
```

Comment:
Wouldn't it make more sense to use the public image here?

Reply:
done